docs/developer-guide/working-with-custom-formatters.md

```diff
@@ -254,48 +296,13 @@ error semi
 
 ### Using a JSON formatter first
```

Comment:
I wonder if we can make this header clearer now? Something that gets across `Outputting to a standard format for other programs to consume`? I don't like that exactly, but does that make sense?